documentation/source/cdm-overview.rst

```diff
@@ -90,12 +90,11 @@ The language includes one default code generator into `Java`_. To facilitate ado
 CDM Applications
 ^^^^^^^^^^^^^^^^
 
-An eco-system of CDM-based application components from software providers exists in order to support the adoption of CDM and the implementation of CDM-based production systems by industry participants. These applications may be open source software or licensed under commercial terms. In particular:
+An ecosystem of CDM-based application components from software providers exists in order to support the adoption of CDM and the implementation of CDM-based production systems by industry participants. These applications may be open source software or licensed under commercial terms. In particular:
 
-* The `CDM Portal`_ provides a user interface allowing users to navigate through and download the CDM.
-* Rosetta is a Software Development Kit (SDK or *dev-kit*) for the Rosetta DSL, that provides an *editable* version of the CDM Portal. It allows the industry community to directly participate in the CDM development by contributing code to the CDM. Please refer to the `Rosetta Documentation`_ for more details.
+* Rosetta is a Software Development Kit (SDK or *dev-kit*) for the Rosetta DSL, that provides xxx industry community to directly participate in the CDM development by contributing code to the CDM. Please refer to the `Rosetta Documentation`_ for more details.
 
-.. note:: The CDM Portal and Rosetta have been developed by technology firm `REGnosys`_. ISDA encourages the adoption of CDM by software providers but does not endorse any CDM application component.
+.. note:: Rosetta have been developed by technology firm `REGnosys`_. ISDA encourages the adoption of CDM by software providers but does not endorse any CDM application component.
```

Comment:
have = has. Also best to remove ISDA from this sentence, perhaps replace with "the CDM community"